ci/prow/e2e-crd-variable

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
```

Comment:
I think this might prevent us from collecting troubleshooting data if one of the commands fails.